Lesson4/AdditionalTask/Document.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace AdditionalTask
+{
+    public class Document
+    {
+        private Title _title;
+        private Body _body;
+        private Footer _footer;
+
+        public Document(Title title, Body body, Footer footer)
+        {
+            _title = title;
+            _body = body;
+            _footer = footer;
+        }
+        public Document()
+        {
+            Initialize();
```

Comment:
Initialize method may be needed rarely for complex logic in constructor or to move repeated code from number of constructors, but simple initializations like here can be placed directly into constructor

Reply:
Got it, thanks)